src/ui/Editor.cpp

```diff
@@ -20,6 +20,7 @@
 #include <SciLexer.h>
 #include <Url.h>
 #include <Volume.h>
+#include <cassert>
```

Comment:
Is this a debug leftover? I can't see any assert call in the diff.
Apart from this, it looks good

Reply:
Ok removed the useless include!